### PR TITLE
Header

### PR DESCRIPTION
**Pull request by v-nakon**
@byzyk

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,50 +1,19 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>test</title>
-	<link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet">
+    <title>static-webpage</title>
 	<link rel="stylesheet" type="text/css" href="style.css">
   </head>
+  <header>
+    <div class="logo"><img src="Logo.png"></div>
```
Comment (byzyk) [inline src/index.html]: Logo should be a text copy written in custom font, not image

File: src/style.css [hunk 2]
```diff
@@ -1,111 +1,40 @@
...
-
+.logo {
+  width: 74.31px;
+  height: 23.18px;
+  left: 125px;
+  top: 59px;
+  position: absolute;
+}
+
+.searchIcon {
```
Comment (byzyk) [inline src/style.css]: it's better to use kebab-case for class names instead of camelCase, for example: `search-icon`

File: src/style.css [hunk 3]
```diff
@@ -1,111 +1,40 @@
...
+  width: 20px;
+  height: 20px;
+  left: 697px;
+  top: 61px;
+  position: absolute;
```
Comment (byzyk) [inline src/style.css]: Don't use position absolute to align elements on the page. It can be done via `flex` instead

File: src/style.css [hunk 4]
```diff
@@ -1,111 +1,40 @@
...
+}
+
+.menu {
+  width: 34px;
+  height: 14px;
+  left: 1201px;
+  top: 64px;
+  position: absolute;
```
Comment (byzyk) [inline src/style.css]: Same as above

File: src/style.css [hunk 5]
```diff
@@ -1,111 +1,40 @@
...
+}
+
+#form {
+  display: none;
+  left: 715px;
+  top: 90px;
+  position: absolute;
```
Comment (byzyk) [inline src/style.css]: same as above

File: src/style.css [hunk 6]
```diff
@@ -1,111 +1,40 @@
...
+}
+
+#toggle {
+  display: none;
+  width: 200px;
+  height: 100%;
+  background-color: red;
+  left: 1001px;
+  top: 64px;
+  position: absolute;
```
Comment (byzyk) [inline src/style.css]: same as above

File: src/index.html [hunk 7]
```diff
@@ -1,50 +1,19 @@
...
 <html>
   <head>
-    <title>test</title>
-	<link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet">
+    <title>static-webpage</title>
 	<link rel="stylesheet" type="text/css" href="style.css">
   </head>
+  <header>
+    <div class="logo"><img src="Logo.png"></div>
+    <div class="searchIcon" id="search"><img src="SearchIcon.png"></div>
```
Comment (byzyk) [inline src/index.html]: it's better to place all images into a separate folder, for example `/images` or `/assets`

File: src/index.html (hunk 1 above)
Comment (v-nakon) [reply src/index.html]: logo don't have font. it's image in template.

File: src/style.css [hunk 8]
```diff
@@ -1,4 +1,10 @@
-.container-header, .container-search, .container-toggle {
+.container {
+  margin: auto;
+  width: 1360px;
+  height: 6738px;
```
Comment (byzyk) [inline src/style.css]: You don't need to set `height` explicitly, it is considered a bad practice unless you need to do it. In our case we want our page to be responsive to the content we'll add later.

File: src/index.html [hunk 9]
```diff
@@ -1,50 +1,32 @@
...
-          </nav>
-	  </div>
+      <div class="container-header">
+        <div class="logo">Gravity</div>
+        <div class="icon-input">
+          
+          <form id="form" class="search-form"> 
+            <input class="search-input"  type="search" placeholder="search">
+          </form>
+          <div class="search-icon" id="search"><img src="images/Search.svg"></div>
```
Comment (byzyk) [inline src/index.html]: `div` is not needed here

File: src/style.css [hunk 10]
```diff
@@ -1,111 +1,57 @@
...
+  position: relative;
+  width: 100px;
+  height: 24px;
+  padding: 59px 0 0 400.12px;
+}
+
+.search-icon {
+  position: absolute;
+  padding-left: 90px;
+  padding-top: 4px;
```
Comment (byzyk) [inline src/style.css]: align icon centrally in %, google how to do it right

File: src/style.css [hunk 11]
```diff
@@ -1,111 +1,57 @@
...
+  font-weight: bold;
+  font-size: 23.18px;
+  padding: 59px 0 0 125px;
+}
+
+.icon-input {
+  position: relative;
+  width: 100px;
+  height: 24px;
+  padding: 59px 0 0 400.12px;
```
Comment (byzyk) [inline src/style.css]: avoid `400.12` values

File: src/style.css [hunk 12]
```diff
@@ -1,111 +1,57 @@
...
+}
+
+.container-header, .container-toggle {
+  display: flex;
+}
+
+.logo {
+  font-family: 'Montserrat', sans-serif;
+  font-weight: bold;
+  font-size: 23.18px;
```
Comment (byzyk) [inline src/style.css]: .18

File: src/index.html [hunk 13]
```diff
@@ -1,50 +1,29 @@
...
-              <li><a href="about.html">About</a></li>
-            </ul>
-          </nav>
-	  </div>
+      <div class="container-header">
+        <div class="logo">Gravity</div>
+        <div class="icon-input">
+          <form id="form" class="search-form"> 
+            <input class="search-input"  type="search" placeholder="search">
+            <button class="button" id="button">Close</button>
```
Comment (byzyk) [inline src/index.html]: move out of form element

File: src/style.css [hunk 14]
```diff
@@ -1,111 +1,115 @@
...
+  position: relative;
+  width: 200px;
+  height: 24px;
+}
+
+.search-icon {
+  position: absolute;
+  left: 50%;
+  margin-left: -20px;
+  margin-top: +3px;
```
Comment (byzyk) [inline src/style.css]: `+` is redundant

File: src/style.css [hunk 15]
```diff
@@ -1,111 +1,115 @@
...
+
+.container-menu {
+  height: 90%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
+
+a {
```
Comment (byzyk) [inline src/style.css]: better define new class than rewriting default tag classes

**Comment by byzyk** [comment]
Good work!

**Pull request merged**